### ISSUE TITLE
Doesn't work anymore

### ISSUE THREAD

**Issue by FDGFGDH** [CodeStix/scoplayer-to-pdf#3]
Hii, quick question. Could somebody please just have a look at this again? because it doesn't work anymore.
Thanks in advance

**Comment by CodeStix** [comment]
Hey, it wasn't working anymore for me too and I fixed it. I'm just not sure its the same issue, can you try again?

**Comment by FDGFGDH** [comment]
I tried it again on several websites, but it still didn't show me the hidden layers.

**Comment by CodeStix** [comment]
Can you send me a link to a website or is it behind authentication? Also, not all websites have hidden layers (or have transparent hidden layers with nothing on them), if you know how, you can check the hidden layers in the chrome dev tools (F12) -> Sources -> look for something interesting.

**Comment by FDGFGDH** [comment]
The websites I use this with are behind authentication, so I can't send them to you. But I know they have a hidden later with content on it since I used them before.

**Comment by FDGFGDH** [comment]
Hey I checked your profile and saw that you are dutch. So I figured you might know this website. https://www.diddit.be/
I mostly used it for this website.

**Comment by CodeStix** [comment]
Yes I do, I actually went to school in belgium and 'knooppunt' (predecessor of diddit) was the inspiration for this extension! I think I can get ahold of some credentials, I will let you know when I know more.

**Comment by FDGFGDH** [comment]
Hey, I'm sorry to bother you again. But I wanted to ask you if you know something yet. I have this dutch assignment and my dutch really isn't good enough for it 😫

**Comment by CodeStix** [comment]
No I don't... But I can help you with the assignment if it has something to do with computer science or the dutch language. :)
